tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compileOnSave": true,
   "compilerOptions": {
-    "target": "es2018",
+    "lib": ["ES2021"],
+    "target": "es2021",
```

Comment:
Bumping this to the latest compatible version with node 18

Reply:
alternatively, it might be easier to use [@tsconfig/node18](https://www.npmjs.com/package/@tsconfig/node18) which actually sets `lib` and `target` to "es2022"…?